### DISCUSSION TITLE
where to see Github as associated account in Github account settings/profile

### DISCUSSION THREAD

**Discussion by AWHubGit** [community/community#52190] [New to GitHub] [answered]
At [https://discourse.gnome.org](https://discourse.gnome.org) Github is connected as an associated account with my email address. I can login by this with Github to [https://discourse.gnome.org](https://discourse.gnome.org).

Where in Github's account settings and profile can I see these allowances I gave to use for logins to other sites?
[https://github.com/settings/profile](https://github.com/settings/profile)

There was once an info but I can't find it again.

**Comment by SMelidoni** [comment, marked as answer]
Hi there @AWHubGit 👋🏽

To see the authorised applications connected to your GitHub account, follow these steps:

1. Log in to your GitHub account.
2. Click on your profile picture in the upper-right corner of the page.
3. Select "Settings" from the dropdown menu.
4. In the left sidebar, click on "Applications".
5. Navigate to the "Authorised OAuth Apps" tab to see the list of applications connected to your GitHub account.

Hope this helps!

**Reply by AWHubGit** [reply]
Yes, this helps. That is I was looking for.

**Reply by SMelidoni** [reply]
Great to hear @AWHubGit! Glad it helped. Would you mind marking my answer as the accepted solution so that others can easily find the solution to this problem? Thank you!

**Reply by Ahmadalbaraka** [reply]
Yeas

**Reply by DazzzlingDasha** [reply]
Does this mean I have to Give out data just to log into GitHub I would be very dissappointed it that is the case

**Comment by Ahmadalbaraka** [comment]
Ahmadalbaraka.com@

**Comment by i-am-anurag** [comment]
Thanks, buddy that's what I was looking for
